test/jdk/java/awt/image/MemoryLeakTest/MemoryLeakTest.java

```diff
@@ -22,154 +22,111 @@
+             ps -al | egrep -i 'java|PPID'
+         Examine the size of the process under SZ.
+         If the two SZ values are the same, plus or minus one,
+         then click Pass, else click Fail.
+        """;
+
+    public static void main(String[] args) throws Exception {
+        PassFailJFrame
+            .builder()
+            .title("MemoryLeakTest Instructions")
+            .instructions(INSTRUCTIONS)
+            .rows(15)
+            .columns(40)
+            .testUI(MemoryLeak::new)
+            .build()
+            .awaitAndCheck();
+        mainThread = Thread.currentThread();
+        if (mainThread != null) {
+            mainThread.interrupt();
+        }
     }
-    else if ( event.getSource() == failButton ) {
-      Globals.testPassed = false;
-      System.err.println(name + " Failed.");
-    }
-    this.dispose();
-    if (Globals.mainThread != null)
-      Globals.mainThread.interrupt();
 }
 
-}
+class MemoryLeak extends Frame implements ComponentListener {
+    private Image osImage;
 
+    public MemoryLeak() {
+        super("Memory Leak Test");
+        setSize(200, 200);
+        addComponentListener(this);
+    }
 
-class MemoryLeak extends Frame implements ComponentListener
-{
-private Image osImage;
+    public static void main(String[] args) {
+        new MemoryLeak().start();
+    }
 
-public MemoryLeak()
-{
-    super("Memory Leak Test");
-    setSize(200, 200);
-    addComponentListener(this);
-}
+    public void start() {
+        setVisible(true);
+    }
 
-public static void main(String args[])
-{
-   new MemoryLeak().start();
-}
+    public void paint(Graphics g) {
+        if (osImage != null) {
+            g.drawImage(osImage, 0, 0, this);
+        }
+    }
 
-public void start()
-{
-  show();
-}
+    public void update(Graphics g) {
+        paint(g);
+    }
 
-public void paint(Graphics g) {
-    if (osImage != null) {
-        g.drawImage(osImage, 0, 0, this);
+    public void componentResized(ComponentEvent e) {
+        Image oldimage = osImage;
+        osImage = createImage(getSize().width, getSize().height);
+        Graphics g = osImage.getGraphics();
+        if (oldimage != null) {
+            g.drawImage(oldimage, 0, 0, getSize().width, getSize().height, this);
+            oldimage.flush();
+        } else {
+            g.setColor(Color.blue);
+            g.drawLine(0, 0, getSize().width, getSize().height);
+        }
+        g.dispose();
     }
-}
 
-public void update(Graphics g)
-{
-    paint(g);
-}
+    public void componentMoved(ComponentEvent e) {}
 
-public void componentResized(ComponentEvent e)
-{
-    Image oldimage = osImage;
-    osImage = createImage(getSize().width, getSize().height);
-    Graphics g = osImage.getGraphics();
-    if (oldimage != null) {
-        g.drawImage(oldimage, 0, 0, getSize().width, getSize().height, this);
-        oldimage.flush();
-    } else {
+    public void componentShown(ComponentEvent e)
+    {
```

Comment:
```suggestion
    public void componentShown(ComponentEvent e) {
```
for the consistency with other code

Reply:
Thanks!